about.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Africa AI summit 2023</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <button class="barbutton" id="barbutton">
+          <i class="fa-sharp fa-solid fa-bars fa-2xl"></i>
+        </button>
+        <ul class="linkto" hidden>
+          <li>
+            <a href="https://github.com/zewdu444/" target="_blank"
+              ><i class="fa-brands fa-github"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://twitter.com/zewdu444" target="_blank"
+              ><i class="fa-brands fa-twitter"></i
+            ></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+        <div class="navwrapper" hidden>
+          <a href="index.html" class="logolink"
+            ><img
+              class="headerlogo"
+              src="./images/ailogo.png"
+              alt="Africa AI conference"
+          /></a>
+          <ul class="navlist" hidden>
+            <li class="about"><a href="about.html">About</a></li>
+            <li><a href="index.html#programs">Program</a></li>
+            <li><a href="index.html#speakers">Speaker</a></li>
+            <li><a href="index.html#partners">Sponsor</a></li>
+            <li><a href="about.html">News</a></li>
+            <li class="aievent"><a href="about.html">AIevent</a></li>
+          </ul>
+        </div>
+        <!-- mobile popup menu -->
+        <div class="popupmenu" id="mobileoption" hidden>
+          <button class="closebutton" id="closebutton">
+            <i class="fa-2xl fa-solid fa-xmark"></i>
+          </button>
+          <ul class="popnavlist">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#programs">Program</a></li>
+            <li><a href="#speakers">Speaker</a></li>
+            <li><a href="#partners">Sponsor</a></li>
+            <li><a href="index.html">News</a></li>
+            <li><a href="index.html">AIevent</a></li>
```

Comment:
**OPTIONAL** - I'd normally advise that it is not necessary to have different nav links for mobile and desktop versions, using CSS and media-queries you have just one set of nav-links, style and hide all elements needed for both mobile and desktop versions. Please fix this 👍 👍. It'd also help to ensure you follow the DRY principle 😄